wire-library/wire-compiler/src/main/java/com/squareup/wire/ConsoleWireLogger.kt

```diff
@@ -31,15 +31,15 @@ internal class ConsoleWireLogger : WireLogger {
     }
   }
 
-  override fun artifactHandled(outputPath: Path, qualifiedName: String) {
+  override fun artifactHandled(outputPath: Path, qualifiedName: String, targetName: String) {
     if (quiet) {
       println(qualifiedName)
     } else {
-      println("Writing $qualifiedName to $outputPath")
+      println("target $targetName writing $qualifiedName to $outputPath")
```

Comment:
I think in this specific case it’s redundant? The output path implies the file type? (We can include this information for other event listeners without using it in the console logger.)